### PR TITLE
UI styling

### PR DESCRIPTION
**Pull request by Tabrisrp**
Add styling for the settings page and the custom admin bar element

### CODE REVIEW

File: src/Admin/Settings/Page.php [hunk 1]
```diff
@@ -228,6 +234,7 @@ public function enqueue_assets( $hook_suffix ) {
 			return;
 		}
 
+		wp_enqueue_style( 'rocketcdn-settings', $this->assets_baseurl . 'css/settings.css', [], '1.0' );
```
Comment (iCaspar) [inline src/Admin/Settings/Page.php:237]: The version param here probably not be hard-coded?

File: src/Admin/Settings/Page.php [hunk 2]
```diff
@@ -228,6 +234,7 @@ public function enqueue_assets( $hook_suffix ) {
...
 		wp_enqueue_script( 'rocketcdn_ajax', $this->assets_baseurl . 'js/ajax.js', [], '1.0', true );
```
Comment (iCaspar) [inline src/Admin/Settings/Page.php:238]: Same here: hardcoded version param.

**Review by iCaspar — CHANGES_REQUESTED** [review]

File: src/Admin/Settings/Page.php (hunk 1 above)
Comment (Tabrisrp) [reply src/Admin/Settings/Page.php:237]: Should we tie the assets version to the plugin version, or each asset have its own version number, only changing when the specific asset changed?

Comment (iCaspar) [reply src/Admin/Settings/Page.php:237]: I think the asset should track the plugin version. If the asset is updated there will need to be a plugin version update as well, and we can always get from version control what the asset looked like with any release.

**Review by iCaspar — APPROVED** [review]

**Pull request merged**